test/test_runner.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Get current test file name
+TESTFILE_NAME=${PGAPPNAME##pg_regress/}
+# Set different name for each test database
+# As pg_regress does not support cleaning db after each test
+TEST_CASE_DB="ldb_${TESTFILE_NAME}"
+
+# Drop db after each test on exit signal
+function drop_db {
+  cat <<EOF | psql "$@" -U postgres -d postgres -v ECHO=none >/dev/null 2>&1
+    SET client_min_messages=ERROR;
+    DROP DATABASE "${TEST_CASE_DB}";
+EOF
+}
+
+trap drop_db EXIT
+
+# install lanterndb extension
+psql "$@" -U postgres -d postgres -v ECHO=none -q -c "DROP DATABASE IF EXISTS ${TEST_CASE_DB};" 2>/dev/null
```

Comment:
Is there a way to not assume `postgres` user? Is the postgres user guaranteed to exist and be accessible via the current shell?
I just want to make sure tests run in the largest number of cases, regardless of people's postgres setup.

Reply:
Yes I will take the user from arguments